lib/std/os/bits/haiku.zig

```diff
@@ -0,0 +1,1410 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2020 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+const std = @import("../../std.zig");
+const maxInt = std.math.maxInt;
+
+// See https://svnweb.freebsd.org/base/head/sys/sys/_types.h?view=co
```

Comment:
This looks like a copy/paste error. (Haiku's equivalent types are defined in `sys/types.h`.)